client/containers/HomePage.js

```diff
@@ -14,7 +14,9 @@ class HomePage extends Component {
   }
 
   handleOnConditionChange = (value) => {
-    this.props.setRadius(value);
+    const { radius, price } = value;
+    this.props.setRadius(radius);
+    this.props.setPrice(price);
```

Comment:
you're saving the price as a string? what if i used it to stuff other than api parameters, does it mean i have to split it every time?